<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>BGI GBS help</title>
<style type="text/css">
body {background-color: #fff; font-size: smaller; margin: 0px 20px; padding-bottom: 20px;}
img{
	border:0px
	}
img,input{
	vertical-align:middle
}
.title {
	font-size: 24px; 
	font-weight: 900;
	}
h1 {font-size: 28px; font-weight: 900; padding-top: 10px; border-top: 5px solid #bbf;}
h2 {font-size: 22px; font-weight: 900;}
h3, .title2 {font-size: 16px; font-weight: 900;}
.title2 {padding-right: 1em;}
p {font-size: 14px; text-indent: 2em;}
img {border: 0px; margin: 10px 0;}
.fig {font-size: 14px;}
.center {text-align: center;}
table {width: 700px; text-align: center; border-top: solid #000 1px; border-right: solid #000 1px; border-bottom: solid #000 1px; margin: 15px 15px 0 0; border-collapse: collapse;}
caption {font-weight: 900;}
tr.head {background-color: rgb(214, 227, 188); text-align: center;}
th, td {border: solid #000 1px; font-size: 12px;}
.listtable {width: 600px;text-align: left;border:none;margin: 15px 15px 0 0;}
.listtable td {text-align: left;border:none;}
.button {padding-right: 1em; padding-left: 1em; display: inline-block; border: 1px solid #000; margin-right: 1em; cursor: pointer; text-align: center; line-height: 1.5em;}
pre {font-size: 12px; padding-left: 2em;}
.note {color: #f00;}
.h3 {font-size: 20px; font-weight: 900; margin-right: 1em;}
.it {font-size: 12px;}
</style>
</head>
<body> 
<a name="top"></a><h1>GBS help</h1>
<div id="report">
<ol style="list-style-type: upper-roman;">
	<li><a href="#it">Bioinformatic analysis</a></li>
	<li><a href="#file">Reference</a></li>
	<li><a href="#list">The list of analysis result</a></li>
</ol>
</div>
<div class="break"></div>
<h2><a name="it">I. Bioinformatic analysis</a></h2>
<div class="title"><span class="h3">1. The flow chart of Bioinformatic analysis</span></div>
<div align="center"><img src="./images/GBS_bio.png" style = "width:600px;"/></div><br />
<div align="center"><span>Figure.1 The flow chart of Bioinformatic analysis</span></div>
</div>
<div class="break"></div>
<div class="title"><span class="h3">2. Sequencing data</span></div>
<p>The original image data of sequencing was converted to sequences in fastq format by base calling. Every read in fastq file was described by four lines as following:</p>
<pre>
@FCB068CABXX:6:1101:1403:2159#TAGGTTAT/1
GTAGAAGACTTATAGATTAAAATTCTCCAACATATAGATGTCCTTACACCGTTTTCCTTTGCTCAGCAGGCTCCGTGTTTGCTTGTCCTT
+
c`bcc_c^ccde_df\c_aeff`ffcfffdfedadca^`b_eed`fe\fed\babdba^Yeebeccfdeae_eec^dbXbda`]bcbebc
</pre>
<p>Line 1 and 3 are names of reads generated by the sequence analyzer(The name after "+" was deleted in order to save storage); Line 2 is sequences; Line 4 is quality values of bases, which equals to ASCII code in line 4 minus 64. For example, the ASCII code of "c" is 99, then its base quality value is 35. The correlation between sequencing error rate and quality values of bases generated by Illumina HiSeq&trade; 2000 was shown in Table 1. If the sequencing error rate is E and Illunima HiSeq&trade; 2000 base quality value is sQ, the equation will be as following:</p>
<div class="center">
sQ = -10log<sub>10</sub>E<br />
<table style="width: 380px; margin: 5px auto;">
	<caption>Tabel.1 Correlation between sequencing error rate and quality value of bases</caption>
	<tr class="head"><th>Sequencing error rate</th><th>Quality values of bases</th><th>Characters</th></tr>
	<tr><td class="center">5%</td><td class="center">13</td><td class="center">M</td></tr>
	<tr><td class="center">1%</td><td class="center">20</td><td class="center">T</td></tr>
	<tr><td class="center">0.1%</td><td class="center">30</td><td class="center">^</td></tr>
</table>
</div>
<div class="title"><span class="h3">3. Data filtering and splitting</span></div>
<ul>
<li{list-style:none;}>1)Remove reads with adaptors;</li><br />
<li{list-style:none;}>2)Remove low quality reads (i.e. reads with more than 50% bases whose quality value are less than or equal to 5);</li><br />
<li{list-style:none;}>3)Classify reads with barcode sequences which verifies between samples;</li><br />
<li{list-style:none;}>4)Remove the reads that cannot be identified with barcode sequnces.</li><br />
</ul>
<div class="title"><span class="h3">4. Reads Alignment or cluster</span></div>
<p>For species with reference genome sequences, clean reads will be mapped against the reference genome sequences with SOAPaligner<sup>[<a href="#cite1">1</a>]</sup>(http://soap.genomics.org.cn).The average depth and coverage will be calculated according to the mapping result. For reads which cannot be mapped, following steps will be taken until the processed reads have alignment on reference sequences: Firstly, remove the first one base and last two bases of read and map it against reference sequences, if the processed reads still can't match on sequences, the last two bases will be removed repeatedly until the alignment is found. The processed reads shorter than 40bp will be removed if there is still no alignment.</p>
<p>For species without reference genome sequences, we will use self-developed NPGT packages or Stacks<sup>[<a href="#cite2">2</a>]</sup>(http://creskolab.uoregon.edu/stacks/)to cluster reads1,and do the next  SNP detection According to the cluster results.<p>
<div class="title"><span class="h3">5. SNP detection</span></div>
<p>For species with reference genome sequences, Considering data characteristics, quality of sequencing and influence of experiments, soapsnp<sup>[<a href="#cite3">3</a>]</sup>will be used for SNP detection, during which a Bayesian model is utilized and every possible genotype probabilities will be calculated based on the observed data. Genotype with the maximum probability will be selected as final genotype and a quality value will be provided to measure the accuracy.
Consensus sequences(Sequences with alignment information on each base) will be generated based on genotypes.Polymorphic loci on reference sequences will be filtered based on consensus sequences. The loci meeting the following criteria will be retained:</p>
<ul>
<li{list-style:none;}>1)Quality value had to be greater than or equal to 20.</li><br />
<li{list-style:none;}>2)Approximate copy number of flanking sequences had to be less than or equal to 1.5.</li><br />
<li{list-style:none;}>3)SNPs had to be at least 5bp away from each other.</li><br />
<li{list-style:none;}>4)Rank sum test statistic had to be greater than or equal to 0.05.</li><br />
</ul>
<p>For species with no reference genome sequences, we use radsnp (a package in NPGT) on reads1 for SNP detection.<p>
<div class="title"><span class="h3">6. Genotying</span></div>
<p>We integrate SNP results of all samples ,filter out the SNP with missing rate greater than 50% and generate genotying list.<p>
<div class="title"><span class="h3">7. Phylogeny analysis</span></div>
<p>Phylogenetic tree (also known as evolutionary tree) is cladogram or tree which describes the evolutionary relationship among groups. The SNPs can be used to calculate the distance between the populations after SNP detection. P (the distance) between the two individuals i and j is calculated by the following formula: <br /><div align="center">D<sub>ij</sub>=/LΣd<sub>ij</sub><sup>(l)</sup>/L (l=1,2,...,L)</p></div>
<p>L is the length of SNPs area. Hypothetically the allele is A/C in position 1, then:</p>
<div align="center">
<table>
<tr><td>d<sub>ij</sub><sup>(l)</sup>=0</td><td>If the genotype of two individuals are AA and AA</td></tr>
<tr><td>d<sub>ij</sub><sup>(l)</sup>=0.5</td><td>If the genotype of two individuals are AA and AC</td></tr>
<tr><td>d<sub>ij</sub><sup>(l)</sup>=0.5</td><td>If the genotype of two individuals are AC and AC</td></tr>
<tr><td>d<sub>ij</sub><sup>(l)</sup>=1</td><td>If the genotype of two individuals are AA and CC</td></tr>
</table>
</div>
<p>We use MEGA5.0<sup>[<a href="#cite4">4</a>]</sup>(http://mega.software.informer.com/5.0/)or TreeBeST(http://treesoft.sourceforge.net/treebest.shtml)or PHYLIP(http://evolution.genetics.washington.edu/phylip.html) software to calculate the distance matrix, based on which we build phylogeny tree using joining method. The bootstrap values is set to 1000.</p>
<div class="title"><span class="h3">8. PCA</span></div>
<p>Principal Component Analysis (PCA) is a mathematical procedure that uses orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables called principal components. We use PCA on autosomal SNP data only and ignore loci with more than two allels or mismatch to reduce interference.</p>
<p>In individual i,d<sub>ik</sub>represents the information about SNP on the site k.If the individual i and reference alleles are homozygous,d<sub>ik</sub>=0;If it is a heterozygote,d<sub>ik</sub>=1;If the individual with the non reference allele is homozygous,d<sub>ik</sub>=2.M is a n * S matrix that include standard genotype:<br /><div align="center"><img src="./images/PCA_matrix_formula.png" style="width:300px"/></div></p>
<p>E(d<sub>ik</sub>)is the average value of d<sub>k</sub>,the individual sample's n×n matrix covariance is calculated in formula: X=MM<sup>T</sup>/S. X's eigenvector decomposition is completed by R's functions.We use EIGENSOFT<sup>[<a href="#cite5">5</a>]</sup>(http://genetics.med.harvard.edu/reich/Reich_Lab/Software.html) built-in package twstats to perform Tracey - Widom test and get the result of the eigenvector significance analysis.</p>
<div class="title"><span class="h3">9. Population structure analysis</span></div>
<p>Population genetic structure refers to a kind of non-random distribution of genetic variation in the species or groups. According to the geographical distribution, or other standards, a group can be divided into several subgroups.Population structure analysis can helps to explain the evolutionary process and found which subgroup the individual belongs to based on the genotype and phenotype correlation research.FRAPPE<sup>[<a href="#cite6">6</a>]</sup>use the maximum likelihood method to interfer each individual's genetic ancestry assuming that the individuals come from the group of the K ancestors.</p>



<h2 class="break"><a name="file"></a>II. Refrence</h2>
<ol>
<li><a name="cite1"></a>Li R.,Yu C.,Li Y.,<i>et al</i>.(2009).SOAP2:An improved ultrafast tool for short read alignment.Bioinformatics,25(15).1966-1967.</li>
<li><a name="cite2"></a>J. Catchen,A. Amores,P. Hohenlohe,W. Cresko, and J. Postlethwait.Stacks:building and genotyping loci denovo from short-read sequences.G3: Genes,Genomes,Genetics,1:171-182,2011.</li>
<li><a name="cite3"></a>Ruiqiang Li, Yingrui Li, Xiaodong Fang,<i>et al</i>.(2009)SNP detection for massively parallel whole-genome resequencing Genome Res.,doi:10.1101/gr.088013.108.</li>
<li><a name="cite4"></a>Koich iro Tamura,<i>et al</i>. MEGA4:Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0.Mol Biol Evol (2007)24(8):1596-1599.doi:10.1093/molbev/msm092.</li>
<li><a name="cite5"></a>Alkes L Price, Patterson NJ,Plenge RM,Weinblatt ME,Shadick NA,Reich D.(2006) Principal components analysis corrects for stratification in genome-wide association studies.Nature Genetics 38,904-909.</li>
<li><a name="cite6"></a>Tang,H.,Peng,J.,Wang,P.& Risch,N.J. Estimation of individual admixture:analytical and study design considerations.Genet.Epidemiol.28,289-301(2005).</li>
</ol>

<h2 class="break"><a name="list"></a>III. The list of analysis result</h2>
<pre>upload<br />|---- Cleandata                     clean reads directory<br />|     |---  A.fq.gz                 sample A clean data<br />|     |---  B.fq.gz                 sample B clean data<br />|---- Data.stat.xls                 all the samples data statistics list<br />|---- SNP.stat.xls                  all the samples SNP statistics list<br />|---- Genotype.xls                  all the samples genotyping statistics list<br />|---- tree                          Phylogenetic tree result directory<br />|     |---  nj_tree.out.tre.png     Phylogenetic tree analysis result (png format)<br />|     |---  nj_tree.out.tre.svg     Phylogenetic tree analysis result(SVG format)<br />|---- PCA                           PCA result directory<br />|     |---  1-2.png                 the contribution rate of component 1 and 2<br />|     |---  1-3.png                 the contribution rate of component 1 and 3<br />|     |---  m-n.png                 the contribution rate of component m and n<br />|---- structure                     Population structure result directory<br />|     |---  STRUCTURE.png           Population structureanalysis result (png format)<br />|     |---  STRUCTURE.svg           Population structureanalysis result (SVG format)<br />|---- index.html                    Chinese version of result report in webpage format <br />|---- index_en.html                 English version of result report in webpage format</pre>
</body>
</html>
