<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>BGI GBS Report</title>
<link href="css/report_en.css" rel="stylesheet" type="text/css">
<script language="javascript" src="js/jquery.js"></script>
<script language="javascript" src="js/report.js"></script>
<style type="text/css">
html{
	overflow:auto;
	}
body{
	position:relative;
	}
caption {font-weight: 900;}
.listtable table {width: 600px;text-align: left;border:none;margin: 15px 15px 0 0;}
.listtable td {text-align: left;border:none;}
p {font-size: 14px; text-indent: 2em;}
</style>
</head>
<body> 
 <div class=seq_cn1>
   <div class="top_btn"></div>
   <h1 class="break"><a name="1"></a>1 Pipeline of experiments</h1>
  <ul>
	<il{list-style:none;}>1)DNA samples,barcode,and common adapter pairs are plated and dried;</li><br />
	<il{list-style:none;}>2)Samples are then digested with restrict enzyme;</li><br />
	<il{list-style:none;}>3)Adapters are ligated to the ends of genomic DNA fragments;</li><br />
	<il{list-style:none;}>4)Pooling and purification;</li><br />
	<il{list-style:none;}>5)PCR: appropriate primers with binding sites on the ligated adapters are added and PCR is performed to increase the fragment pool;</li><br />
	<il{list-style:none;}>6-7)Cleaned up PCR products, checked fragment sizes of the resulting library on a DNA analyzer.And sequence DNA.</li><br />
	<ul>
	<div class="tc"><img src="./images/GBS_exp.png"  style="width: 650px;" alt="" /></div><br />
  <div class="tc">Figure.1 Pipeline of experiments</div>
<h1 class="break"><a name="2"></a>2 Bioinformatics analysis</h1><br /><div align="center"><img src="./images/GBS_bio.png" style = "width:600px;"/></div><br /><div align="center"><span>Figure.2 The flow chart of Bioinformatic analysis</span></div><h2 class="break"><a name="2.1"></a>2.1 Data statistics</h2><p>52.61 Gb clean data was generated in this project. Data statistics of some samples was shown as following (Please read <a href="./readme/Data.readme_en.txt" target="_blank">Data.readme_en.txt</a> for definition of each column.):</p><center><table><caption>Table.1 Data statistics(Clean_Data)</caption><tr class="head"><th>Sample name</th><th>Read number (M)</th><th>Base number (Mb)</th><th>GC (%)</th><th>Q20 (%)</th><th>Q30 (%)</th></tr><tr><td>1HL_10A</td><td>6.37</td><td>615.01</td><td>42.61</td><td>95.68</td><td>90.80</tr><tr style="height: 5px;"><td>1HL_11A</td><td>7.14</td><td>688.72</td><td>42.69</td><td>95.72</td><td>90.91</tr><tr style="height: 5px;"><td>1HL_12A</td><td>5.05</td><td>487.29</td><td>42.57</td><td>95.61</td><td>90.70</tr><tr style="height: 5px;"><td>1HL_13A</td><td>5.35</td><td>516.55</td><td>42.84</td><td>95.63</td><td>90.72</tr><tr style="height: 5px;"><td>1HL_14A</td><td>4.03</td><td>387.00</td><td>42.69</td><td>95.59</td><td>90.64</tr><tr style="height: 5px;"><td>1HL_15A</td><td>4.39</td><td>421.69</td><td>42.50</td><td>95.59</td><td>90.62</tr><tr style="height: 5px;"><td>1HL_16A</td><td>4.67</td><td>447.97</td><td>42.62</td><td>95.63</td><td>90.74</tr><tr style="height: 5px;"><td>1HL_17A</td><td>7.31</td><td>701.52</td><td>42.51</td><td>95.61</td><td>90.70</tr><tr style="height: 5px;"><td>1HL_19A</td><td>7.24</td><td>694.65</td><td>42.76</td><td>95.64</td><td>90.73</tr><tr style="height: 5px;"><td>1HL_1A</td><td>4.52</td><td>436.26</td><td>42.36</td><td>95.61</td><td>90.66</tr><tr style="height: 5px;"><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</tr></table><div style="width: 600px; text-align: left;">*Data statistics of all samples:<a href="./Data.stat.xls" target="_blank">Data.stat.xls</a></div><br /></center><h2 class="break"><a name="2.2"></a>2.2 SNP Detection</h2><p>The information about SNPs in one sample was shown as following(Please read <a href="./readme/SNP.readme_en.txt" target="_blank">SNP.readme_en.txt</a> for definition of each column.):</p><center><table><caption>Table.2 SNP statistics</caption><tr class="head"><th>Sample name</th><th>Total</th><th>Homo</th><th>Hete</th><th>Homo rate (%)</th><th>Hete rate (%)</th></tr><tr><td>1HL_10A</td><td>8058</td><td>6755</td><td>1303</td><td>83.83</td><td>16.17</tr><tr style="height: 5px;"><td>1HL_11A</td><td>8392</td><td>7033</td><td>1359</td><td>83.81</td><td>16.19</tr><tr style="height: 5px;"><td>1HL_12A</td><td>7422</td><td>6405</td><td>1017</td><td>86.30</td><td>13.70</tr><tr style="height: 5px;"><td>1HL_13A</td><td>7580</td><td>6499</td><td>1081</td><td>85.74</td><td>14.26</tr><tr style="height: 5px;"><td>1HL_14A</td><td>6374</td><td>5552</td><td>822</td><td>87.10</td><td>12.90</tr><tr style="height: 5px;"><td>1HL_15A</td><td>6683</td><td>5706</td><td>977</td><td>85.38</td><td>14.62</tr><tr style="height: 5px;"><td>1HL_16A</td><td>7041</td><td>6043</td><td>998</td><td>85.83</td><td>14.17</tr><tr style="height: 5px;"><td>1HL_17A</td><td>8419</td><td>7050</td><td>1369</td><td>83.74</td><td>16.26</tr><tr style="height: 5px;"><td>1HL_19A</td><td>8543</td><td>7141</td><td>1402</td><td>83.59</td><td>16.41</tr><tr style="height: 5px;"><td>1HL_1A</td><td>6919</td><td>5949</td><td>970</td><td>85.98</td><td>14.02</tr><tr style="height: 5px;"><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</tr></table><div style="width: 600px; text-align: left;">* SNPs in all samples:<a href="./SNP.stat.xls" target="_blank">SNP.stat.xls</a></div><br /></center><h2 class="break"><a name="2.3"></a>2.3 Genotyping</h2><p>The genotyping result was shown as following (Please read <a href="./readme/genotype.noref.readme_en.txt" target="_blank">genotype.noref.readme_en.txt</a> for definition of each column.):</p><center><table><caption>Table.3 Genetying result-noref</caption><tr class="head"><th>ID</th><th>Consensus_Seq</th><th>pos</th><th>1SN_9A</th><th>1SN_8A</th><th>1SN_7A</th><tr style="height: 5px;"><td>record_385</td><td>CAGCCCTTATTAGGCCACCTGAGTRAACTGATGTGACCTATATTGTAATTAGTTTTCATCCATCATTGTTAAGTCATATGTA</td><td>25</td><td>A</td><td>A</td><td>A</tr><tr style="height: 5px;"><td>record_795</td><td>CAGCTTCCGATGCGTCGGAGAAGATCAGCACTTYTGGATCTGATACTAAGCTCAGTGATTGTGGAACAAACATCCGTGGAAT</td><td>34</td><td>Y</td><td>Y</td><td>Y</tr><tr style="height: 5px;"><td>record_1907</td><td>CAGCTGAAGCATCCAGCAAAGCACGAACAAAGCAGAGAGACACCTACAACACCAAAGTGAGARGAGGAACGGTTAAGGCTGG</td><td>63</td><td>G</td><td>G</td><td>-</tr><tr style="height: 5px;"><td>record_2412</td><td>CAGCCCTTCACCGGTAATGGTGACGTCTCAATATGAATGAAATATTCTCGACGGGWCGAAAAACAACAAATAATCAATCAAT</td><td>56</td><td>-</td><td>-</td><td>-</tr><tr style="height: 5px;"><td>record_3293</td><td>CAGCACCTCAGGGCTATCCGTTTCTAACACGGGTACGGGCCMGCGGGGGTTCACGGGCAGTGACCTCCCTTGCACGTTACAA</td><td>42</td><td>-</td><td>C</td><td>-</tr><tr style="height: 5px;"><td>record_4526</td><td>CTGCACTTGACCACAATAAATGTGATCTTCCTTTAGATTTAATATATATCTGTCTGCCTTCATRAACGTTGATTTTTAAAAT</td><td>64</td><td>R</td><td>G</td><td>R</tr><tr style="height: 5px;"><td>record_4591</td><td>CAGCTGTATCATCTGACCCCAGAAGAAGCCGGYTTGATAGGAAGCTCTCTTGATGTGTGCTTCTCGAGTTGGAGGAATGGCC</td><td>33</td><td>C</td><td>C</td><td>C</tr><tr style="height: 5px;"><td>record_5127</td><td>CTGCGTTGTCATGCAAGCATTCTTGACTCCCATGCTCACGAGCCATAGTTACAATGCCGGAATCGCRCATTGGTTTATTCGC</td><td>67</td><td>R</td><td>G</td><td>R</tr><tr style="height: 5px;"><td>record_5132</td><td>CAGCTTAAGATTTTGTGCATTTTTCAGTTGTGYTCTCTCTACTTTCGATTCCTTGATCCGCCCTTGAAGGAGACCTTCATGT</td><td>33</td><td>C</td><td>C</td><td>-</tr><tr style="height: 5px;"><td>record_5456</td><td>CAGCTTGTTTACGAGGCCCTGCYCGCACTTTCTACATAAGTCTCCCCGCAGAATTCAGACAGCCATATAACTTGTTAATTAC</td><td>23</td><td>Y</td><td>Y</td><td>Y</tr><tr style="height: 5px;"><td>...</td><td width="15%">...</td><td width="15%">...</td><td width="15%">...</td><td width="15%">...</td><td width="15%">...</td></tr></table><div style="width: 600px; text-align: left;">* Genotyping result of all samples:<a href="./Genotype.xls" target="_blank">Genotype.xls</a></div><br /></center><h2 class="break"><a name="2.4"></a>2.4 Phylogeny analysis</h2><p>According to the SNPs we got, the evolution relationship between groups can be infered and shown in phylogenetic tree as following:</p><div class="tc"><img src="./tree/nj_tree.out.tre.png" style="width: 600px;" alt="" /></div><div class="tc">Figure.3 Phylogenetic tree.</div><br /><br /><div style="clear: both;"></div><h2 class="break"><a name="2.5"></a>2.5 PCA</h2><p>PCA (Principal Component Analysis) is a mathematical procedure that uses orthogonal transformation to convert a set of observations of possibly correlated variables into a set of values of linearly uncorrelated variables call principal components, which have the largest possible variances. Based on different SNPs between samples, we could separate samples into different subgroups by PCA. The analysis result was shown below:</p><div class="tc"><img src="./PCA/1_2.png" style="width: 600px;" alt="" /></div><div class="tc">Figure.4 PCA.</div><p class="note" align="center" style="color:red">Note:Dots in different color represent in different subgroups.</p></div>
</body>
</html>  
